# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1406916485 14400
#      Fri Aug 01 14:08:05 2014 -0400
# Node ID fe505ff21d5bf00553ee452631dccc4ee89d1c4c
# Parent  dcb260e7a6487e6001bba12d5c0e10b82598861f
# Parent  7761c281aac09e9ccf5c0b82404366c31aa74597
maint: Periodic merge of gui-release to default.

diff --git a/libinterp/parse-tree/module.mk b/libinterp/parse-tree/module.mk
--- a/libinterp/parse-tree/module.mk
+++ b/libinterp/parse-tree/module.mk
@@ -86,17 +86,17 @@ PARSE_TREE_SRC = \
 parse-tree/oct-gperf.h: parse-tree/octave.gperf
 	$(GPERF) -t -C -D -G -L C++ -Z octave_kw_hash $< > $@-t1
 	$(SED) 's,lookup\[,gperf_lookup[,' < $@-t1 > $@-t
 	mv $@-t $@
 	rm -f $@-t1
 
 parse-tree/oct-parse.yy: parse-tree/oct-parse.in.yy
 	case "$(BISON_API_PREFIX_DECL_STYLE)" in \
-	  *api*) api_prefix_decl='%define api.prefix "octave-"'; ;; \
+	  *api*) api_prefix_decl='%define api.prefix "octave_"'; ;; \
 	  *name*) api_prefix_decl='%name-prefix="octave_"'; ;; \
 	esac; \
 	case "$(BISON_PUSH_PULL_DECL_STYLE)" in \
 	  *quote*) quote='"' ;; \
 	  *) quote="" ;; \
 	esac; \
 	case "$(BISON_PUSH_PULL_DECL_STYLE)" in \
 	  *dash*) push_pull_decl="%define api.push-pull $${quote}both$${quote}"; ;; \
